### PR TITLE
add 10-POST-comment

### PR DESCRIPTION
**Pull request by Alex-J-Jackson**

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -264,3 +264,42 @@ describe("GET /api/reviews/:review_id/comments", () => {
...
+  });
+  test("returns 404 for usernames not in database", () => {
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send({ username: "not-a-user", body: "test-body" })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("ID or user not found");
+      });
+  });
```
Comment (HeatherEver) [inline __tests__/index.test.js:304]: couple more tests to add - what is there was an empty comment body? 
if they entered an invalid article_id

**Review by HeatherEver — COMMENTED** [review]
nice just a couple more tests to add
